scalafmt-core/shared/src/main/scala/org/scalafmt/internal/FormatOps.scala

```diff
@@ -194,6 +194,15 @@ class FormatOps(val tree: Tree, val initStyle: ScalafmtConfig) {
   def nextNonComment(curr: FormatToken): FormatToken =
     nextNonCommentWithCount(curr)._2
 
+  def nextNonBlank(curr: FormatToken): FormatToken = {
+    if (!(curr.right.is[Space] || curr.right.is[Tab])) curr
```

Comment:
Can this ever return true? `FormatToken.{right,left}` are always non-whitespace.

Reply:
Right, I've made that fix before knowing it, so I'm sure there's a better way. I'll try.